src/Dynamo.All.sln

```diff
@@ -235,6 +235,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "PythonMigrationViewExtensio
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "NodeAutoCompleteViewExtension", "NodeAutoCompleteViewExtension\NodeAutoCompleteViewExtension.csproj", "{51511AFD-F326-4995-8E27-5D711419EF6F}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Md2Html", "Tools\MD2HTML\Md2Html.csproj", "{0893F745-CB1A-427A-8E87-CA927273039A}"
```

Comment:
if this tool is cross platform does it make sense to add it to the mono.sln as well?

Reply:
I will give it a try and see if it works in mono.

Reply:
Mono do compiles so keep for now. I will address your other cross platform comments next.